packages/components/popper/src/use-popper/build-modifiers.ts

```diff
@@ -37,7 +37,10 @@ export default function buildModifier(
       name: 'flip',
       options: {
         padding: 5,
-        fallbackPlacements: fallbackPlacements ?? [],
+        fallbackPlacements:
```

Comment:
I think it is not the reason here, refer to the other components

Reply:
I see. Adding a `fallback-placements` attribute value to the `autocomplete` component can fix this problem.